UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -990,4 +990,11 @@ public void CmdDoEmote(string emoteName)
 	{
 		EmoteActionManager.DoEmote(emoteName, playerScript.gameObject);
 	}
+
+	[Command]
+	public void CmdResetMovementForSelf()
+	{
+		playerScript.playerMove.ResetEverything();
```

Comment:
what does this do?

Reply:
Basically gets rid of things like sliding, flying, etc and resets movement to a non-moving state. An issue I found while working on #9816 is that when the player gets launched out, they are still considered flying/sliding which causes the game to prevent the player from moving. 
If other issues follow a similar pattern, we will now know that the issue is from how we handle all these variables that are in the movement script and narrow it down there.